tests/libs/baseAction.test.ts

```diff
@@ -240,4 +200,34 @@ describe("BaseAction", () => {
 
     await expect(baseAction["getPrivateKey"]()).rejects.toThrow("process exited");
   });
+
+  describe("formatOutput", () => {
+    test("should return string as is", () => {
+      expect((baseAction as any).formatOutput("Hello")).toBe("Hello");
+    });
+
+    test("should format an object using util.inspect", () => {
+      const data = {key: "value", num: 42};
+      const result = (baseAction as any).formatOutput(data);
+      expect(result).toBe(inspect(data, {depth: null, colors: false}));
+    });
+
+    test("should format an error object using util.inspect", () => {
+      const error = new Error("Test Error");
+      const result = (baseAction as any).formatOutput(error);
+      expect(result).toBe(inspect(error, {depth: null, colors: false}));
+    });
+
+    test("should format a Map object using util.inspect", () => {
+      const testMap = new Map([["key1", "value1"]]);
+      const result = (baseAction as any).formatOutput(testMap);
+      expect(result).toBe(inspect(testMap, {depth: null, colors: false}));
+    });
+
+    test("should format a BigInt object using util.inspect", () => {
+      const bigIntValue = BigInt(9007199254740991);
+      const result = (baseAction as any).formatOutput(bigIntValue);
+      expect(result).toBe(inspect(bigIntValue, {depth: null, colors: false}));
+    });
```

Comment:
@epsjunior if you can set the exact expected output without the inspect, it would be ideal.
Something like: 
`expect(result).toBe({key1: "value1"});`